File: convert_indentation.py
import re

with open('src/components/Presentation.tsx', 'r', encoding='utf-8') as f:
    content = f.read()

# Convert 2-space indentation to 4-space indentation
lines = content.split('\n')
converted_lines = []
for line in lines:
    # Count leading spaces
    leading_spaces = len(line) - len(line.lstrip(' '))
    if leading_spaces > 0 and leading_spaces % 2 == 0:
        # Convert 2-space indentation to 4-space
        new_leading_spaces = (leading_spaces // 2) * 4
        converted_line = ' ' * new_leading_spaces + line.lstrip(' ')
        converted_lines.append(converted_line)
    else:
        converted_lines.append(line)

with open('src/components/Presentation.tsx', 'w', encoding='utf-8') as f:
    f.write('\n'.join(converted_lines))

print("Conversion complete: 2-space indentation converted to 4-space indentation")

File: convert_indentation.js
const fs = require('fs');

// Read the file
const content = fs.readFileSync('src/components/Presentation.tsx', 'utf8');

// Convert 2-space indentation to 4-space indentation
const lines = content.split('\n');
const convertedLines = lines.map(line => {
    // Count leading spaces
    const leadingSpaces = line.match(/^ */)[0].length;
    if (leadingSpaces > 0 && leadingSpaces % 2 === 0) {
        // Convert 2-space indentation to 4-space
        const newLeadingSpaces = (leadingSpaces / 2) * 4;
        return ' '.repeat(newLeadingSpaces) + line.trimStart();
    }
    return line;
});

// Write the converted content back
fs.writeFileSync('src/components/Presentation.tsx', convertedLines.join('\n'));

console.log('Conversion complete: 2-space indentation converted to 4-space indentation');
